foliant/preprocessors/base.py

```diff
@@ -25,36 +25,13 @@ def get_options(options_string: str) -> Dict[str, OptionValue]:
         if not options_string:
             return {}
 
-        def _cast_value(value: str) -> OptionValue:
-            '''Attempt to convert a string to integer, float, or boolean.
-
-            If nothing matches, return the original string.
-
-            :param value: String to convert
-
-            :returns: Converted value or original string
-            '''
-
-            value = value.strip('"')
-
-            try:
-                return int(value)
-            except ValueError:
-                try:
-                    return float(value)
-                except ValueError:
-                    try:
-                        return bool(strtobool(value))
-                    except ValueError:
-                        return value
-
         option_pattern = re.compile(
-            r'(?P<key>[A-Za-z_:][0-9A-Za-z_:\-\.]*)="(?P<value>.+?)"',
+            r'(?P<key>[A-Za-z_:][0-9A-Za-z_:\-\.]*)=(\'|")(?P<value>.+?)\2',
```

Comment:
How about to assign a name to the group? It seems more beautiful.

Reply:
I did it intentionally, because the quote group is only needed for capturing quotes, not to use in the code afterwards. For example, if we call the `groupdict` method of match object, now we will get following result:

```python
{'key': 'param', 'value': 'param_value"}
```

If we name the group we will get

```python
{'key': 'param', 'quote': '"', 'value': 'param_value"}
```

which may be confusing for developer